### PR TITLE
app-admin/mcelog: Fix call to undeclared library function strncpy on …

### PR DESCRIPTION
**Pull request by listout**
…musl

Closes: https://bugs.gentoo.org/894316

### CODE REVIEW

File: app-admin/mcelog/mcelog-194-r1.ebuild [hunk 1]
```diff
@@ -13,7 +13,7 @@ SRC_URI="https://github.com/andikleen/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64 x86"
```
Comment (thesamesam) [inline app-admin/mcelog/mcelog-194-r1.ebuild:16]: You're losing stable keywords here? Either copy it to a new one and drop stable keywords, or git mv and keep the stable keywords.

Comment (listout) [reply app-admin/mcelog/mcelog-194-r1.ebuild:16]: doing, soory. forgot about pkgbump

Comment (thesamesam) [reply app-admin/mcelog/mcelog-194-r1.ebuild:16]: `pkgcheck scan --commits` should flag this as `DroppedStableKeywords` too

Comment (listout) [reply app-admin/mcelog/mcelog-194-r1.ebuild]: is it not recommended to rm the old ebuild when a new revision is made?

**Pull request closed without merging**